common/on_action/wc_exodar_on_action.txt

```diff
@@ -0,0 +1,69 @@
+﻿#### Yearly check to see if Exodar event chain can begin
+yearly_global_pulse = {
+	on_actions = { exodar_check }
+}
+exodar_check = {
+	effect = {
+		if = {
+			limit = { NOT = { exists = global_var:exodar_event_chain_started } }
+			random = {
+				chance = 0 # No chance to start for all bookmarks
+				modifier = {
+					add = 2 # 2% chance per year
+					current_date >= 606.1.1
+				}
+				modifier = {
+					add = 8 # 10% chance per year
+					current_date >= 610.1.1
+				}
+				modifier = {
+					add = 15 # 25% chance per year
+					current_date >= 620.1.1
+				}
+				set_global_variable = {
+					name = exodar_event_chain_started
+					value = yes
+				}
+				trigger_event = { on_action = exodar_setup months = { 2 4 } }
+			}
+		}
+	}
+}
+
+#### Exodar Setup, and Early Warning to Azuremyst Isle Players
+exodar_setup = {
+	effect = {
+		every_player = {
+			limit = { capital_province ?= { geographical_region = world_kalimdor_north_veiled_isles } }
```

Comment:
i think it shouldn't be capital province : /

Reply:
Done.